etc/profile-m-z/prismlauncher.profile

```diff
@@ -0,0 +1,34 @@
+# Custom profile for prismlauncher
+
+# file system
+include /etc/firejail/disable-common.inc
+whitelist ~/.local/share/PrismLauncher
+read-only ~/Downloads
+include /etc/firejail/whitelist-common.inc
+private-tmp
+private-dev
+disable-mnt
+blacklist /mnt
+blacklist /media
+blacklist /sbin
+
+# network
+net enp4s0f3u2u1u2
```

Comment:
The ne of the interface is a detail of *your* hardware.

Reply:
I adjusted the profile to match the template